modules/packages/LinearAlgebra.chpl

```diff
@@ -437,45 +493,45 @@ proc _array.T where isDefaultRectangularArr(this) && this.domain.rank == 2
   return transpose(this);
 }
 
-/* Add matrices, maintaining dimensions, deprecated for ``_array.plus`` */
+/* Element-wise addition. Deprecated for ``A + B`` */
 proc matPlus(A: [?Adom] ?eltType, B: [?Bdom] eltType) where isDefaultRectangularArr(A) && isDefaultRectangularArr(B) {
-  compilerWarning('matPlus has been deprecated for _array.plus, ' +
-                  'try: A.plus(B)');
+  compilerWarning('matPlus has been deprecated. ' +
+                  'try: A + B');
   return A.plus(B);
 }
 
-/* Add matrices, maintaining dimensions */
+/* Element-wise addition. Same as ``A + B``. */
 proc _array.plus(A: [?Adom] ?eltType) where isDefaultRectangularArr(A) && isDefaultRectangularArr(this) {
```

Comment:
Seems unfortunate to have `_array` show up in user-facing docs. Not sure if there's anything we can do about that short of another post-processing script.

Reply:
Agreed. Probably another post-processing script or `chpldoc` addition.